resources/i18n/qqq.json

```diff
@@ -39,6 +39,8 @@
 	"videos": "{{identical|Video}}",
 	"yes": "{{identical|yes}}",
 	"no": "{{identical|no}}",
+	"add-bookmark": "Represents the action of adding a page as a bookmark",
+	"remove-bookmark": "Represents the action of removing a page from the bookmarks",
```

Comment:
Keys in `qqq.json` must match those in `en.json`. Actually these key names are better than the ones in `en.json`.